core/common/util/src/test/java/org/eclipse/dataspaceconnector/common/configuration/ConfigurationFunctionsTest.java

```diff
@@ -64,23 +73,20 @@ public void returnSystemProperty(String key, String expected) {
     @SetEnvironmentVariable(key = ENV_VAR_1, value = VALUE_1)
     @SetEnvironmentVariable(key = ENV_VAR_2, value = "")
     @SetEnvironmentVariable(key = EDC_VAR_3, value = "    ")
+    @SetEnvironmentVariable(key = EDC_VAR_4, value = VALUE_1 + "    ")
     @MethodSource("envVarsSource")
     public void returnEnv(String key, String expected) {
         String resultValue = ConfigurationFunctions.propOrEnv(key, DEFAULT);
         assertThat(resultValue).isEqualTo(expected);
     }
 
-    @Test
```

Comment:
My preference is for explicit tests rather than moving this to the parameterized tests because it verifies a specific case.

Reply:
Both variants cover the exact same thing, therefore it is more about which version you personally prefer. But i am fine with splitting it up in a separate test.